.github/dependabot.yaml

```diff
@@ -59,38 +124,6 @@ updates:
           - "black"
           - "djlint"
     ignore:
-      # Temporarily pinned to v2.14.0 - problem for translations found in v2.15.0, see: https://github.com/ONSdigital/eq-questionnaire-runner/pull/1384
+      # "black" temporarily pinned to v2.14.0 - problem for translations found in v2.15.0, see: https://github.com/ONSdigital/eq-questionnaire-runner/pull/1384
```

Comment:
`"black"` -> `"babel"`

Reply:
Done, thanks, might be because I am doing four different tasks simultaneously.